packages/frontend/src/components/Strategies/Crab/StrategyPnLV2.tsx

```diff
@@ -0,0 +1,168 @@
+import { Box, CircularProgress, IconButton, Typography,  Hidden, TextField, Tooltip,  InputAdornment, } from '@material-ui/core'
+import { createStyles, makeStyles } from '@material-ui/core/styles'
+import React, { memo, useState } from 'react'
+import dynamic from 'next/dynamic'
+import useAppMemo from '@hooks/useAppMemo'
+import { crabV2DaysAtom, useCrabPnLV2ChartData } from 'src/state/ethPriceCharts/atoms'
+import InfoIcon from '@material-ui/icons/InfoOutlined'
+import { atom, useAtom, useAtomValue } from 'jotai'
+import { graphOptions } from '@constants/diagram'
+import LegendBox from '@components/LegendBox'
+
+
+export type ChartDataInfo = {
+    timestamp: number
+    ethUsd: number
+    crabEth: number 
+    crabUsd: number
+    crabPnL: number
+  }
+
+
+enum ChartType {
+    PNL = 'PNL',
+}
+const chartTradeTypeAtom = atom(0)
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    container: {
+       
+        padding: theme.spacing(0),
+      marginTop: theme.spacing(4),
+      marginBottom: theme.spacing(5),
+      maxWidth: '640px',
+    },
+    green: {
+      color: theme.palette.success.main,
+    },
+    red: {
+      color: theme.palette.error.main,
+    },
+    navDiv: {
+        display: 'flex',
+        marginBottom: theme.spacing(2),
+        alignItems: 'center',
+        marginTop: theme.spacing(2),
+      },
+    legendBox: {
+        display: 'flex',
+        gap: '10px',
+        marginTop: '10px',
+        justifyContent: 'center',
+      },
+      payoffContainer: {
+        
+        //  background: theme.palette.background.stone,
+         borderRadius: theme.spacing(1),
+        padding: theme.spacing(1, 2),
+        display: 'flex',
+        marginTop: theme.spacing(2),
+        [theme.breakpoints.up('sm')]: {
+          maxHeight: '310px',
+        },
+        [theme.breakpoints.down('sm')]: {
+          flexDirection: 'column',
+        },
+       
+      },
+  }),
+)
+const Chart = dynamic(() => import('kaktana-react-lightweight-charts'), { ssr: false })
+
+const modeAtom = atom<ChartType>((get) => {
```

Comment:
If the mode is not going to be updated and used anywhere else, can we move it to a variable inside the component wrapped in useMemo?

Reply:
So the intention is to add several chart types, Crab/USD,  Crab/ETH since we have the data....but to cut down scope..we decided to implement for only Crab Pnl and added the other features to the backlog